### ISSUE TITLE
[Suggestion] Support multiple items for auto-on item

### ISSUE THREAD

**Issue by IJustWantANamePlease** [SchmarrnDevs/Lighty#68]
So... I read from the changelog that the auto-on item is not plural form (also I tried to make a list which just crashes the game), so I guess it's not possible right now?
It would be really great and convenient to have this feature since there are too many light sources in the game.

**Comment by andi-makes** [comment]
Thanks for the suggestion!

Yeah, I do want to make it a list, but that requires some extra work. I just wanted to push the feature out there, to get some feedback/testing (just like your feedback, thanks again!) before polishing/finishing the feature entirely.

So, TL;DR: Yes, I plan on making it a kind of list! Just a little bit more work to do :smiley:

**Comment by comfysquirrel** [comment]
I'd also find this feature very useful!

**Comment by andi-makes** [comment]
I've implemented this in Lighty v3.0.0-beta.4, with the new default items being the torches and lanterns of Minecraft.

Because I've kept the changes backwards compatible, the new defaults won't be applied to existing configs.

To change it, do the following:

edit `lighty/base.config` and change the line with `lighty.auto_on.item` to say the following:

```config
lighty.auto_on.item=minecraft:torch minecraft:soul_torch minecraft:lantern minecraft:soul_lantern
```

The syntax is `lighty.auto_on.item=modid:item modid:item modid:item`, so after the equals sign one ResourceLocation followed by exactly one space character followed by another ResourceLocation. Repeat as often as you need. There shouldn't be a trailing space character at the end or after the =, this will break the config format.

I plan on making some kind of wiki, documenting this and any other config options that aren't configurable inside of the game, but I hope that for now this is sufficient enough information on how to configure this feature.

Thanks for the patience, feel free to reopen this issue if there are any more questions regarding the config format. If there are issues with the feature in general, please open a new issue.

Thanks and have fun!
